Core/Core/Views/EmptyViewController.swift

```diff
@@ -19,22 +19,23 @@
 import UIKit
 
 public class EmptyViewController: UIViewController {
-    @objc var showLogo: Bool = true
+    var showLogo: Bool = true
     override public func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
         if(showLogo) { addLogo() }
     }
-    
-    @objc func addLogo() {
+
+    func addLogo() {
         let image = UIImage(named: "EmptyViewControllerLogo", in: .core, compatibleWith: nil)
```

Comment:
could we use the `.icon(.instructure)` instead of a dedicated resource for this?